recipes/mic.lua

```diff
@@ -0,0 +1,164 @@
+--[[
+
+        Licensed under GNU General Public License v2
+        * (c) 2021, bzgec
+
+
+# Microphone state widget/watcher
+This widget can be used to display the current microphone status.
+
+## Requirements
+- `amixer` - this command is used to get and toggle microphone state
+
+## Usage
+- Download [mic.lua](https://awesomewm.org/recipes/mic.lua) file and put it into awesome's
+  folder (like `~/.config/awesome/widgets/mic.lua`)
+- Add widget to `theme.lua`:
+
+  ```lua
```

Comment:
Same issue as in #156: There are spaces before the code block beginning

(I'll not look at this further; let's first do #156)

Again: Thanks for the contributions!